internal/pod_models_test.go

```diff
@@ -940,6 +963,76 @@ var _ = Describe("pod_models", func() {
 			})
 		})
 
+		Context("with a different process group crash looping in main container", func() {
+			BeforeEach(func() {
+				cluster.Spec.Buggify.CrashLoopContainers = []fdbv1beta2.CrashLoopContainerObject{
+					{
+						ContainerName: fdbv1beta2.MainContainerName,
+						Targets:       []string{"storage-2"},
+					},
+				}
+				spec, err = GetPodSpec(cluster, fdbv1beta2.ProcessClassStorage, 1)
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			It("should have the normal start command", func() {
+				mainContainer := spec.Containers[0]
+				Expect(mainContainer.Name).To(Equal(fdbv1beta2.MainContainerName))
+				Expect(mainContainer.Args).To(Equal([]string{
+					"fdbmonitor --conffile /var/dynamic-conf/fdbmonitor.conf" +
+						" --lockfile /var/dynamic-conf/fdbmonitor.lockfile" +
+						" --loggroup operator-test-1" +
+						" >> /var/log/fdb-trace-logs/fdbmonitor-$(date '+%Y-%m-%d').log 2>&1",
+				}))
+			})
+		})
+
+		Context("with a different process group crash looping in container", func() {
```

Comment:
This test is the same as the test above `with an instance that is crash looping container` correct?

Reply:
More or less. One is checking Command and another is checking Args field. Just followed the pattern of existing crash loop test.